tutorials/6_nodecg-configuration.md

```diff
@@ -18,6 +18,7 @@ NodeCG is configured via a `cfg/nodecg.json` file with the following schema:
 - `bundles` _Object_ Contains configuration for bundles.
   - `enabled` _Array of strings_ A whitelist array of bundle names that will be the only ones loaded at startup. Cannot be used with `bundles.disabled`.
   - `disabled` _Array of strings_ A blacklist array of bundle names that will not be loaded at startup. Cannot be used with `bundles.enabled`.
+  - `paths` _Array of strings_ An array of additional paths where bundles are located.
```

Comment:
Can this be clearer in terms of if this path has to be the directory that contains bundles or the directory of the bundle itself?